src/components/button/button.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+const Button = ({ className, text, onClick }) => {
```

Comment:
We've already discussed about this component, please remove it or change that this component does